_dev/sources/addin.ts

```diff
@@ -430,6 +499,7 @@ class Addin {
         this.distributionListsBuilder.unload();
         this.miscBuilder.unload();
         this.exportBtn.removeEventListener("click", this.exportData, false);
+        this.saveBtn.removeEventListener("click", this.saveChanges, false);
```

Comment:
Add removeEventListener methods to checkboxes.

Reply:
done.